src/sentry/eventstore/models.py

```diff
@@ -1,22 +1,30 @@
 from __future__ import absolute_import
 
+import pytz
+
 from datetime import datetime
-from django.utils import timezone
+from dateutil.parser import parse as parse_date
 
 from sentry.models import EventCommon, EventDict
 from sentry.db.models import NodeData
 
+from sentry.snuba.events import Columns
+
 
 def ref_func(x):
     return x.project_id or x.project.id
 
 
 class Event(EventCommon):
-    def __init__(self, project_id, event_id, group_id=None, data=None):
+    def __init__(
+        self, project_id, event_id, group_id=None, message=None, data=None, snuba_data=None
+    ):
         self.project_id = project_id
         self.event_id = event_id
         self.group_id = group_id
+        self.message = message
```

Comment:
why was this promoted to a field of the Event class ?

Reply:
The EventManager sets the `event.message` property to be something else (the full search message) than what event.message is the rest of the time :/ The Django event also had a top level message field that held this value as well. It might be better to give this a distinct name like `search_message` or something.